pkg/osquery/tables/dev_table_tooling/table_test.go

```diff
@@ -0,0 +1,69 @@
+package dev_table_tooling
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"encoding/base64"
+	"testing"
+
+	"github.com/go-kit/kit/log"
+	"github.com/kolide/launcher/pkg/osquery/tables/tablehelpers"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_generate(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		name           string
+		commandName    []string
```

Comment:
Why is this an array? Is it because (in theory) we could pass multiple commands? If so, `commandNames`

Reply:
Good catch, no need for an array here